docs/source/getting-started/app-architecture.rst

```diff
@@ -0,0 +1,108 @@
+.. Copyright (c) 2020 The DAML Authors. All rights reserved.
+.. SPDX-License-Identifier: Apache-2.0
+
+App Architecture
+****************
+
+In this section we'll look at the different components of our social network app.
+The goal is to familiarise you enough to feel comfortable extending the code with a new feature in the next section.
+
+There are two main components in the code - the DAML model and the React/TypeScript frontend - with generated TypeScript code to bridge the two.
+Let's start by looking at the DAML model, as this sets the core logic of the application.
+
+The DAML Model
+==============
+
+Using VSCode (or a code editor of your choice), navigate to the ``daml`` subdirectory.
```

Comment:
We want people to have a pleasant UX. Currently, this only works with VS Code. Let's not suggest to use a different editor in the main text for now. We could add a footnote but I'm not sure we need to.